ui2/src/components/deployment/new/SelectService.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import { TableTransfer } from '../../../common/TableTransfer';
+import { Spinner } from '../../../common/Spinner';
+
+export const SelectService = ({ getServices, services, handleBreadcrumbs, getServicesStack, servicesStacks, resetBreadcrumbs, match }) => {
```

Comment:
i think the same logic we talk about the selected service/env/region
could be applied to the service stack as well 
you ask for the service stack when enter the page from the redux 
he return them if they are there if not he will fetch them